gotpm/cmd/seal_test.go

```diff
@@ -84,17 +95,23 @@ func TestUnsealFail(t *testing.T) {
 	rwc := internal.GetTPM(t)
 	defer tpm2tools.CheckedClose(t, rwc)
 	ExternalTPM = rwc
+	extendPCR := func(rw io.ReadWriter, pcr int) error {
+		pcrToExtend := tpmutil.Handle(pcr)
+		extension := bytes.Repeat([]byte{0xAA}, sha256.Size)
+		return tpm2.PCRExtend(rw, pcrToExtend, tpm2.AlgSHA256, extension, "")
+	}
 
 	tests := []struct {
-		name    string
-		tpmFunc func(io.ReadWriter) error
+		name             string
+		sealPCRs         string
+		certifyPCRs      string
+		tpmFunc          func(io.ReadWriter, int) error
+		tpmFuncParameter int
 	}{
 		// TODO(joerichey): Add test that TPM2_Reset make unsealing fail
-		{"ExtendPCR", func(rw io.ReadWriter) error {
-			pcrToExtend := tpmutil.Handle(23)
-			extension := bytes.Repeat([]byte{0xAA}, sha256.Size)
-			return tpm2.PCRExtend(rw, pcrToExtend, tpm2.AlgSHA256, extension, "")
-		}},
+		{"ExtendSealPCR", "23", "", extendPCR, 23},
```

Comment:
The PCR is extended, so call it ExtendPCRAndSeal or something like that.

Reply:
ExtendPCRAndUnseal
ExtendPCRAndCertify
ExtendPCRAndCertify2